pkg/dnfjson/dnfjson.go

```diff
@@ -315,6 +317,7 @@ type repoConfig struct {
 	SSLClientKey   string   `json:"sslclientkey,omitempty"`
 	SSLClientCert  string   `json:"sslclientcert,omitempty"`
 	MetadataExpire string   `json:"metadata_expire,omitempty"`
+	ModuleHotfixes *bool    `json:"module_hotfixes"`
```

Comment:
This should be marked as `omitempty`. It will be marshaled to JSON as `null` value and then unmarshaled by `dnf-json` as `None`, which will most probably result in problems in `dnf-json`...

Reply:
oh totally, missed that!